fixtures/pages-functions-wasm-app/functions/other-meaning-of-life.ts

```diff
@@ -0,0 +1,5 @@
+import { CHOCOLATE } from "../chocolate";
```

Comment:
What's this testing? Just confirming that regular bundling works fine?

Reply:
yes, that's correct. Since we never bundled _worker.js before, I wanted to make sure we have that covered in a fixture

Reply:
Understood, but this is a `functions/` dir, yes? You don't have a fixture test checking non-Wasm imports in a `_worker.js`, (but you have covered it extensively in regular tests so absolutely not necessary imo!)

Reply:
lol...ignore me, I'm a pinhead 🙄